js/views/modals/purchase/Payment.js

```diff
@@ -114,15 +123,27 @@ export default class extends BaseVw {
   walletConfirm() {
     this.getCachedEl('.js-payFromWallet').addClass('processing');
     this.spendConfirmBox.setState({ show: false });
+    const currency = getServerCurrency().code;
 
     try {
       spend({
         address: this.paymentAddress,
         amount: this.balanceRemaining,
-        currency: getServerCurrency().code,
+        currency,
       })
+        .done(() => {
+          recordEvent('walletSpend', {
+            currency,
+            err: '',
```

Comment:
We seem to be using error in other countly events. Can we be consistent and either use "err" or "error" consistently? I think it would make for a cleaner review of the countly data.